yacc/console/gram.y

```diff
@@ -362,10 +361,14 @@ trace_stmt:
 	{
 		$$ = &TraceStmt{All: true}
 	} | 
-	START TRACE CLIENT any_id {
+	START TRACE CLIENTID any_id {
```

Comment:
why CLIENTID?

Reply:
All other cmds uses CLIENT, like KILL CLIENT or SHOW ... WHERE CLIEND

Reply:
It conflicts with other commands. When CLIENT is in reserved_keyword.go commands like kill client do not work, and when CLIENT is not in the list START TRACE CLIENT does not work

Reply:
If you have any idea how to fix this it would be great